packages/gatsby/src/bootstrap/load-plugins/utils/check-local-plugin.ts

```diff
@@ -0,0 +1,41 @@
+import { PluginRef } from "../types"
```

Comment:
This is the same logic as that that existed before but flattened with early returns instead of nested conditionals